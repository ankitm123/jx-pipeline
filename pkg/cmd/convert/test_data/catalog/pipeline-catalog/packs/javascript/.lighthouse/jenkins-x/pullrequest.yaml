apiVersion: tekton.dev/v1
kind: PipelineRun
metadata:
  annotations:
    lighthouse.jenkins-x.io/prependStepsURL: https://raw.githubusercontent.com/jenkins-x/jx3-pipeline-catalog/60bed6408732c1eda91a15713f51a9f97dcb1757/tasks/git-clone/git-clone-pr.yaml
  name: pullrequest
spec:
  pipelineSpec:
    tasks:
    - name: from-build-pack
      computeResources: {}
      taskSpec:
        metadata: {}
        stepTemplate:
          name: ""
          computeResources:
            requests:
              cpu: 400m
              memory: 512Mi
          workingDir: /workspace/source
        steps:
        - image: gcr.io/jenkinsxio/jx-boot:3.1.156
          name: jx-variables
          computeResources: {}
          script: |
            #!/usr/bin/env sh
            jx gitops variables
        - image: node:12-slim
          name: build-npm-install
          computeResources: {}
          script: |
            #!/bin/sh
            npm install
        - image: node:12-slim
          name: build-npm-test
          computeResources: {}
          script: |
            #!/bin/sh
            CI=true DISPLAY=:99 npm test
        - image: gcr.io/kaniko-project/executor:debug-v1.3.0
          name: build-container-build
          computeResources: {}
          script: |
            #!/busybox/sh
            source .jx/variables.sh
            cp /tekton/creds-secrets/tekton-container-registry-auth/.dockerconfigjson /kaniko/.docker/config.json
            /kaniko/executor $KANIKO_FLAGS --context=/workspace/source --dockerfile=Dockerfile --destination=$DOCKER_REGISTRY/$DOCKER_REGISTRY_ORG/$APP_NAME:$VERSION
        - image: gcr.io/jenkinsxio/jx-preview:0.0.143
          name: promote-jx-preview
          computeResources: {}
          script: |
            #!/usr/bin/env sh
            source .jx/variables.sh
            jx preview create

  taskRunTemplate:
    serviceAccountName: tekton-bot
  timeouts:
    pipeline: 240h0m0s
