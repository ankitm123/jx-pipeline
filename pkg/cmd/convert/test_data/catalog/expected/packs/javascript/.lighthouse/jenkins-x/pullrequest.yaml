apiVersion: tekton.dev/v1
kind: PipelineRun
metadata:
  creationTimestamp: null
  name: pullrequest
spec:
  pipelineSpec:
    tasks:
    - name: from-build-pack
      taskSpec:
        metadata: {}
        spec: null
        stepTemplate:
          computeResources:
            requests:
              cpu: 400m
              memory: 512Mi
          image: uses:jenkins-x/jx3-pipeline-catalog/tasks/javascript/pullrequest.yaml@versionStream
          workingDir: /workspace/source
        steps:
        - computeResources: {}
          image: uses:jenkins-x/jx3-pipeline-catalog/tasks/git-clone/git-clone-pr.yaml@versionStream
          name: ""
        - computeResources: {}
          name: jx-variables
        - computeResources: {}
          name: build-npm-install
        - computeResources: {}
          name: build-npm-test
        - computeResources: {}
          name: build-container-build
        - computeResources: {}
          name: promote-jx-preview
  taskRunTemplate:
    serviceAccountName: tekton-bot
  timeouts:
    pipeline: 240h0m0s
status: {}
